webpack/react_app/components/TargetingHosts/components/HostStatus.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Icon } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const HostStatus = ({ status }) => {
+  switch (status) {
+    case 'cancelled':
+      return (
+        <div>
+          <Icon type="pf" name="warning-triangle-o" /> {status}
+        </div>
+      );
+    case 'N/A':
+      return (
+        <div>
+          <Icon type="fa" name="question" /> {status}
+        </div>
+      );
+    case 'running':
+      return (
+        <div>
+          <Icon type="pf" name="running" /> {status}
+        </div>
+      );
+    case 'planned':
+      return (
+        <div>
+          <Icon type="pf" name="build" /> {status}
+        </div>
+      );
+    case 'warning':
+      return (
+        <div>
+          <Icon type="pf" name="error-circle-o" /> {__('failed')}
+        </div>
+      );
+    case 'error':
+      return (
+        <div>
+          <Icon type="pf" name="error-circle-o" /> {__('failed')}
+        </div>
+      );
```

Comment:
because they are the same I think you can do something like:
```suggestion
    case 'warning':
￼    case 'error':
￼      return (
￼        <div>
￼          <Icon type="pf" name="error-circle-o" /> {__('failed')}
￼        </div>
￼      );
```